lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -17,6 +17,7 @@
 use Doctrine\DBAL\Schema\Column;
 use Doctrine\DBAL\Schema\ColumnDiff;
 use Doctrine\DBAL\Schema\Constraint;
+use Doctrine\DBAL\Schema\Expressions\Expression;
```

Comment:
The `Expressions` piece seems redundant.

Reply:
fixed